Demo/SwiftUI Viper Demo /DemoApp/Modules/Home/HomeView.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import SwiftUI
+
+struct HomeView: View {
+
+    @ObservedObject var presenter: HomePresenter
+    @State private var isRotated: Bool = false
```

Comment:
Can we have a @Published in the Presenter instead of State in view for a demo app please 😅 
(It can be some other property, my comment is more about showcasing the presenter than this property)

Reply:
I have added Login module which showcases the Published property from Presenter. But I can add it in this module as well, if neccessary